util/env.go

```diff
@@ -1,10 +1,42 @@
 package util
 
-import "os"
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+	"os"
+)
 
 // GetConfigValue get the environment value using the key.
 // if not found, then fetches it from AWS Parameter Store
 func GetConfigValue(key string) string {
 	//TODO: include AWS Parameter Store
-	return os.Getenv(key)
+	value := os.Getenv(key)
+	if value != "" {
+		return value
+	}
+	prefix := os.Getenv("SSM_PS_NP")
+	region := os.Getenv("SSM_PS_RG")
+	sess, err := session.NewSessionWithOptions(session.Options{
```

Comment:
@abhishek246 Can you make the **sess** and **ssmsvc** objects as local variable on this file?